conjure-lib/src/main/java/com/palantir/conjure/java/lib/internal/ConjureClientService.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.lib.internal;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Signifies that the annotated class is a conjure service.
+ */
+@Retention(RetentionPolicy.CLASS)
+@Target(ElementType.TYPE)
+public @interface ConjureClientService {
+    String package_() default "";
```

Comment:
Is this not always just the package/class name of the annotated class?

Reply:
local codegen plugin applies a different package prefix.

Reply:
it may be, but, as carter would say, that's fragile to assume. I'd rather have it explicit. Carter also wants to not have this annotation at all and just use the Endpoint annotation. I have the client side code working with just the Endpoint information, so maybe that is good enough, even though I like having the service annotation for reference when reading the source code.